test/integration/crud/crud_api.test.ts

```diff
@@ -670,21 +670,7 @@ describe('CRUD API', function () {
         // Legacy update method
         // -------------------------------------------------
         const legacyRemove = function () {
-          db.collection('t4_1').insertMany(
```

Comment:
Removing the `single` option necessitates refactoring this test to use `deleteOne`.  There is already a test for `deleteOne` specifically, and we're removing `remove` in another PR, so removing this block of code felt like the correct approach.